lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -73,6 +74,12 @@ void _hideAutofillElements(html.HtmlElement domElement) {
     ..textShadow = 'transparent'
     ..transformOrigin = '0 0 0';
 
+  if (outsideOfScreen) {
+    elementStyle
+      ..top = '-50px'
+      ..left = '-50px';
```

Comment:
Is this enough? In other places we use `-9999px`.

Reply:
change. -50px also worked though :)